.eslintrc.json

```diff
@@ -5,8 +5,9 @@
     "extends": "standard",
     "rules": {
         "semi": [2, "always"],
-        "indent": ["error", 4],
-        "mocha/no-exclusive-tests": "error"
+        "indent": 0,
+        "mocha/no-exclusive-tests": "error",
+        "space-before-function-paren": 0
```

Comment:
This conflicts with prettier formatting which consistently **doesn't** add the space.